cli/internal/cmd/init.go

```diff
@@ -45,7 +45,7 @@ func NewInitCmd() *cobra.Command {
 			cmd.Flags().Bool("yes", false, "")
 			// Don't skip any phases
 			// The apply backend should handle init calls correctly
```

Comment:
Can you fix the comment for this.
I think eventually we just want to remove the changes from https://github.com/edgelesssys/constellation/pull/2457
But that can wait until we merge `create` into `apply`.